src/cpu-template-helper/src/template/verify/mod.rs

```diff
@@ -22,8 +22,7 @@ pub use x86_64::verify;
 pub enum VerifyError {
     /// {0} not found in CPU configuration.
     KeyNotFound(String),
-    #[rustfmt::skip]
-    #[doc = "Value for {0} mismatched.\n{1}"]
+    /// Value for {0} mismatched.\n{1}
```

Comment:
Ok, the last thing. As it appears `#[doc = "Value for {0} mismatched.\n{1}"]` and `/// Value for {0} mismatched.\n{1}` are not equal. The later converts `\n` into `\\n`. So in this case let's use the `/** */`. Never though Rust comments with new lines will be such a pain.
```diff
-    /// Value for {0} mismatched.\n{1}
+    /** Value for {0} mismatched.
+    {1} */
```

Reply:
Haha strange. Thanks for the help.